pkg/internal/api/constants.go

```diff
@@ -7,6 +7,7 @@ const (
 
 	ReleaseChannelDefault = "default"
 	ReleaseChannelFast    = "fast"
+	ReleaseChannelSlow    = "slow"
```

Comment:
Not sure how valuable this is now, and might be out of scope for this PR but I see values in approaches like this to make sure we only accept the defined values:

```go
type ReleaseChannel string

const (
    ReleaseChannelDefault ReleaseChannel = "default"
    ReleaseChannelFast    ReleaseChannel = "fast"
    ReleaseChannelSlow    ReleaseChannel = "slow"
)
```